core/src/filed/fd_plugins.cc

```diff
@@ -2096,6 +2096,9 @@ static bRC bareosSetValue(PluginContext* ctx, bVariable var, void* value)
   switch (var) {
     case bVarSinceTime:
       jcr->fd_impl->since_time = (*(int*)value);
+      jcr->fd_impl->incremental = (jcr->fd_impl->since_time != 0);
+      SetFindOptions(jcr->fd_impl->ff, jcr->fd_impl->incremental,
+                     jcr->fd_impl->since_time);
```

Comment:
I think this messes up some of bareos internal logic.  The job should not change its "level" after its been started.
I.e. if you had a fileset with three include blocks where the first and third one have normal files and the second one has a plugin that sets incremental to true.

If you now run a full backup the first include block will be a normal full but the third one will behave like an incremental i believe.